testcase.py

```diff
@@ -645,6 +645,7 @@ def _get_var_metadata(self, fmu, var_list, inputs=False):
                     maxi = None
             var_metadata[var] = {'Unit':unit,
                                  'Description':description,
+                                 'tags': self.tags_json.get(var),
```

Comment:
Does it make sense to capitalize ``'tags'`` like other meta data?

Reply:
certainly. good catch.